tests/it/util.rs

```diff
@@ -0,0 +1,119 @@
+use std::{
+    ffi::{OsStr, OsString},
+    path::{Path, PathBuf},
+};
+
+use assert_cmd::{assert::Assert, cargo::CargoError, Command};
+use assert_fs::{prelude::PathChild, TempDir};
+
+pub fn wasm_file(name: &str) -> PathBuf {
+    let path = Path::new(name);
+    let wasm = if path
+        .extension()
+        .map_or(false, |ext| ext.eq_ignore_ascii_case("wasm"))
+    {
+        ""
+    } else {
+        ".wasm"
+    };
```

Comment:
❗ Let's make this work one way, and expect the .wasm to always be present or not. I don't think we need to support both.